Example/MoyaComparison/MoyaComparison/Utils/MCLogger+LogModule.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  MCLogger+LogModule.swift
+//  MoyaComparison
+//
+//  Created by Steven_WATREMEZ on 30/03/2017.
+//  Copyright © 2017 niji. All rights reserved.
+//
+
+import Foundation
+import SwiftyBeaver
+
+public enum LogModule: String {
+  case appCoordinator
+  case githubProvider
+  case webServiceClient
+  case parsing
+  case encoding
+}
+
+extension SwiftyBeaver {
```

Comment:
I personally now don't do an extension on `SwiftyBeaver` anymore.

Instead, I declare the functions below as instance methods of the `enum LogModule`, so that it's not class methods anymore, I don't have to pass the `LogModule` as a parameter (because it's now `self`), and it reads nicer at call site.

So for example:
```swift
extension LogModule {
  public func debug(_ message: @autoclosure @escaping () -> Any, _ path: String = #file, _ function: String = #function, line: Int = #line) {
    SwiftyBeaver.debug({ return "[\(self.rawValue)] - \(message())" as Any}(), path, function, line: line)
  }
  // same for the verbose(…), info(…), warning(…) etc methods
}
```
And at call site it will look like:
```swift
LogModule.webServiceClient.debug("Hello")
// instead of:
// SwiftyBeaver.debug(.webServiceClient, "Hello")
```